### DISCUSSION TITLE
How to update Python Version in Pipfile, Pipfile.lock *and* Dockerfile in one PR?

### DISCUSSION THREAD

**Discussion by sbrandtb** [renovatebot/renovate#8777] [Request Help]
Let's suppose we are using Pipenv with `python_version: 3.8`, have a Pipfile.lock that inevitably also has that constraint and a Dockerfile with `FROM python:3.8-slim`.

With default settings, renovate opened a merge request that updates the Docker image, but not the Pipenv part, and our builds fail.

Ideally, renovate should update the version in Dockerfile, Pipfile and maintain Pipfile.lock in the same MR.

**Comment by rarkins** [comment]
To upgrade them together means they need to have the same branch name. Usually the easiest way to do that is creating a package rule matching the packages (eg by name) and including a "groupName" setting.

**Reply by rarkins** [reply]
Also set `"additionalBranchPrefix": ""` in the same rule
